api/rosetta/status.go

```diff
@@ -67,9 +73,6 @@ func (d *Data) Status(ctx echo.Context) error {
 		return echo.NewHTTPError(http.StatusInternalServerError, Internal(err))
 	}
 
-	// TODO: See if it makes sense to include the genesis block information:
-	// => https://github.com/optakt/flow-dps/issues/152
```

Comment:
There should be an issue to actually implement the genesis block return, though :D